src/attached_probe.cpp

```diff
@@ -527,7 +527,12 @@ void AttachedProbe::load_prog()
     if (bt_verbose) {
       std::cerr << std::endl << "Error log: " << std::endl << log_buf << std::endl;
       if (errno == ENOSPC) {
-        std::cerr << "Error: No space left on device, try increasing BPFTRACE_LOG_SIZE environment variable" << std::endl;
+        // TODO: Can we extract the current value of bpftrace.log_size_ here to
+        // print it out?
```

Comment:
Good idea - you can get it here from `probe_.log_size`

Reply:
Nice!  That helped a lot.  Done.